microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -21,9 +21,13 @@ This tutorial covers the following extension technologies:
 
 You'll learn to update your pop-up menu to replace your static stars image with a title and a standard HTML button.  That button, when selected, passes that image of stars to the content page.  This image is now embedded in the extension and inserted into the active browser tab. Here are the steps.
 
-## Step 1: Remove the image from the pop-up and replace it with a button.
+Note: These steps require that you have completed the steps for initial extension package steps for the tutorial, go to [MicrosoftEdge-Extensions repo > extension-getting-started-part1](/microsoft-edge/extensions-chromium/getting-started/part1-simple-extension?tabs=v3).  
```

Comment:
"These steps require that you complete the steps for initial extension package steps. For the tutorial, go to [MicrosoftEdge-Extensions repo > extension-getting-started-part1](/microsoft-edge/extensions-chromium/getting-started/part1-simple-extension?tabs=v3). " 

This would be more clear.

Reply:
fixed in the repo and will resubmit.